connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectSessionBasedSuite.scala

```diff
@@ -32,7 +35,9 @@ trait SparkConnectPlanTestWithSparkSession extends SharedSparkSession with Spark
   override def getSession(): SparkSession = spark
 }
 
-class SparkConnectDeduplicateSuite extends SparkConnectPlanTestWithSparkSession {
+class SparkConnectSessionBasedSuite extends SparkConnectPlanTestWithSparkSession {
```

Comment:
so we still have 2 different dsl test suites?

Reply:
I am working on a refactoring PR but that is blocked. Let's have a discussion on how to unblock it.

With the refactoring I expect we only have one suite that test Connect proto with DataFrame API.

Before that is done, right now we have two suites, one is with DataFrame API and another one is with Catalyst.